tools/main-transpile-wrapper.js

```diff
@@ -0,0 +1,8 @@
+// This file exists because it is the file in the tool that is not automatically
+// transpiled by Babel.
+
+require('source-map-support').install();
```

Comment:
Because it installs a global handler, basically. So you only get one per
process. And we do depend on the other one for stack traces in the tool
(less importantly, if things like ddp loaded into the tool crash; more
importantly, if things like user plugin code throws).
On Jun 11, 2015 12:57 AM, "Sashko Stubailo" <notifications@github.com>
wrote:

> In tools/main-transpile-wrapper.js
> <https://github.com/meteor/meteor/pull/4533#discussion_r32197311>:
>
> > @@ -0,0 +1,8 @@
> > +// This file exists because it is the file in the tool that is not automatically
> > +// transpiled by Babel.
> > +
> > +require('source-map-support').install();
>
> I will investigate more and figure out what to do about this.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/meteor/meteor/pull/4533/files#r32197311>.
>